src/app/containers/MostWatched/index.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState, useEffect, useContext } from 'react';
+import 'isomorphic-fetch';
+import { arrayOf, shape, bool, string } from 'prop-types';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+
+import { storyItem } from '#models/propTypes/storyItem';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { RequestContext } from '#contexts/RequestContext';
+import useToggle from '#hooks/useToggle';
+import StoryPromo from '../StoryPromo';
+import Grid from '../../components/Grid';
+import CpsOnwardJourney from '../CpsOnwardJourney';
+import { getMostWatchedData, processMostWatched } from './utilities';
+
+const MostWatched = ({ initialData, hasHeader }) => {
+  const { dir, mostWatched, service } = useContext(ServiceContext);
+  const { variant, isAmp } = useContext(RequestContext);
+
+  const defaultMostWatchedConfig = {
+    header: 'Most Watched',
+    numberOfItems: 10,
+    hasMostWatched: false,
```

Comment:
We need to look into how we can use toggles for this, rather than service config

Reply:
Done